examples/example-gateway/build/endpoints/register.go

```diff
@@ -17,89 +14,89 @@ import (
 	"github.com/uber/zanzibar/runtime"
 )
 
-type handlerFn func(
-	ctx context.Context,
-	req *zanzibar.ServerHTTPRequest,
-	res *zanzibar.ServerHTTPResponse,
-	clients *clients.Clients,
-)
-
-type myEndpoint struct {
-	HandlerFn handlerFn
-	Clients   *clients.Clients
-}
-
-func (endpoint *myEndpoint) handle(
-	ctx context.Context,
-	req *zanzibar.ServerHTTPRequest,
-	res *zanzibar.ServerHTTPResponse,
-) {
-	fn := endpoint.HandlerFn
-	fn(ctx, req, res, endpoint.Clients)
+// Endpoints is a struct that holds all the endpoints
+type Endpoints struct {
+	ArgNotStructHandler     *bar.ArgNotStructHandler
+	MissingArgHandler       *bar.MissingArgHandler
+	NoRequestHandler        *bar.NoRequestHandler
+	NormalHandler           *bar.NormalHandler
+	TooManyArgsHandler      *bar.TooManyArgsHandler
+	CallHandler             *baz.CallHandler
+	SimpleHandler           *baz.SimpleHandler
+	SimpleFutureHandler     *baz.SimpleFutureHandler
+	SaveContactsHandler     *contacts.SaveContactsHandler
+	AddCredentialsHandler   *googlenow.AddCredentialsHandler
+	CheckCredentialsHandler *googlenow.CheckCredentialsHandler
 }
 
-func makeEndpoint(
-	g *zanzibar.Gateway,
-	endpointName string,
-	handlerName string,
-	middlewares []zanzibar.MiddlewareHandle,
-	handlerFn handlerFn,
-) *zanzibar.Endpoint {
-	myEndpoint := &myEndpoint{
-		Clients:   g.Clients.(*clients.Clients),
-		HandlerFn: handlerFn,
+// CreateEndpoints bootstraps the endpoints.
+func CreateEndpoints(
+	gateway *zanzibar.Gateway,
+) interface{} {
+	return &Endpoints{
+		ArgNotStructHandler: bar.
+			NewArgNotStructEndpoint(gateway),
+		MissingArgHandler: bar.
+			NewMissingArgEndpoint(gateway),
+		NoRequestHandler: bar.
```

Comment:
We cant line break before "bar", gofmt will put it back on one line. 

I'll just keep it all on one line for now.